app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I dont really know what it does, it was there when i made it and ive read up about it but dont really understand it